tests/system_test.py

```diff
@@ -578,6 +584,15 @@ def __enter__(self):
     def __exit__(self, *_):
         self.teardown()
 
+    @property
+    def qd_manager(self):
+        if not self._qd_manager:
+            try:
+                self._qd_manager = QdManager(address=self.addresses[0])
+            except Exception as e:
+                return None
```

Comment:
Probably don't want to silently discard the exception - it will be very hard to figure out what happened if the QdManager fails in GH actions.

Reply:
> Probably don't want to silently discard the exception - it will be very hard to figure out what happened if the QdManager fails in GH actions.

You mean, it will be hard to figure out why the QdManager constructor fails in GHA ? The only one reason it could fail is if there is no self.addresses, right ? I will add a print(e) before returning None

Reply:
Is not having a self.addresses[0] a bug, or is it OK as long as the caller checks for a None?  I think that's my point of confusion: I don't understand why the exception is being ignored. If the API allows a None return code (not a bug), then I would simply comment the code to explain that is why the exception is discarded.

Reply:
As it stands now, not all routers have a valid self.addresses[0], so it is ok to return a None. I will add a comment clarifying that. But the person who fixes https://github.com/skupperproject/skupper-router/issues/753 will make not having a self.addresses[0] illegal and require every router to have a management port on self.addresses[0]